### PR TITLE
Updated core files to older version

### PR DESCRIPTION
**Pull request by vrlnx**
Major change, I do not know the outcome for your versions, but for the people that tried this with more go it to work.

Next pull request will be a overhaul on the startup.sh with core installation for end-user.

No edit is recommended. Everything should be spoken about in advance.

Changelog:
pip > pip3

win-x32 | ubuntu:16.04 > ubuntu:14.04

-vrlnx

### CODE REVIEW

**Comment by malwaredllc** [comment]
@vrlnx Thanks for your work on this! I made some notes in the code review for you to take a look at when you have a chance.

**Comment by vrlnx** [comment]
Where do I go to see these comments?

File: web-gui/startup.sh [hunk 1]
```diff
@@ -21,14 +21,14 @@ status=$?
...
-	apt-get install docker.io -y
+	apt install docker.io -y
 else
 	echo "Confirmed Docker is installed."
 fi
 
 # Install Python packages
 echo "Installing Python packages..."
-python3 -m pip install -r requirements.txt
+pip3 install -r requirements.txt
```
[the remark this answers is not in the document]
Comment (cclauss) [reply web-gui/startup.sh:31]: https://snarky.ca/why-you-should-use-python-m-pip/

File: web-gui/docker-pyinstaller1/Dockerfile-py3-win32 [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:16.04
+FROM ubuntu:14.04
```
[the remark this answers is not in the document]
Comment (vrlnx) [reply web-gui/docker-pyinstaller1/Dockerfile-py3-win32:1]: I did a 20.04 test first, but Docker could not create win32 nor nix docker images becaouse of this. When I tried 14.04, that worked like a charm with Web gui interface to generate windows exe payloads

File: byob/crosscomp/build_environment.sh [hunk 3]
```diff
@@ -206,8 +206,8 @@ make_containers() {
 FROM $xc_dockimg
 COPY $str_binfmt $map_binfmt
 CMD ls ./ /usr/bin
-RUN /usr/bin/apt-get update
-RUN /usr/bin/apt-get install -y upx-ucl build-essential python python-pip zlib1g-dev python3 python3-pip python3-wheel python3-setuptools python3-dev python3-distutils python3-venv
+RUN /usr/bin/apt update
+RUN /usr/bin/apt install -y upx-ucl build-essential python python-pip zlib1g-dev python3 python3-pip python3-wheel python3-setuptools python3-dev python3-distutils python3-venv
```
[the remark this answers is not in the document]
Comment (vrlnx) [reply byob/crosscomp/build_environment.sh:210]: I always use apt, I'm trying to abandon apt-get as much as I can. You can read more about it here: https://phoenixnap.com/kb/apt-vs-apt-get

File: web-gui/startup.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (vrlnx) [reply web-gui/startup.sh:31]: Ok, I did not know this! I'll start using python3 -m pip then. But when I did use it. Some of the pip3 modules was never installed. I don't know why.

File: web-gui/docker-pyinstaller1/entrypoint-windows.sh [hunk 4]
```diff
@@ -19,14 +19,14 @@ if [[ "$PYPI_URL" != "https://pypi.python.org/" ]] || \
...
-    echo "trusted-host = $(echo $PYPI_URL | perl -pe 's|^.*?://(.*?)(:.*?)?/.*$|$1|')" >> /wine/drive_c/users/root/pip/pip.ini
+    mkdir -p /wine/drive_c/users~/pip
+    echo "[global]" > /wine/drive_c/users~/pip/pip.ini
+    echo "index = $PYPI_URL" >> /wine/drive_c/users~/pip/pip.ini
+    echo "index-url = $PYPI_INDEX_URL" >> /wine/drive_c/users~/pip/pip.ini
+    echo "trusted-host = $(echo $PYPI_URL | perl -pe 's|^.*?://(.*?)(:.*?)?/.*$|$1|')" >> /wine/drive_c/users~/pip/pip.ini
 
     echo "Using custom pip.ini: "
-    cat /wine/drive_c/users/root/pip/pip.ini
+    cat /wine/drive_c/users~/pip/pip.ini
```
[the remark this answers is not in the document]
Comment (vrlnx) [reply web-gui/docker-pyinstaller1/entrypoint-windows.sh:29]: I have had no issues running exe, maybe you know something I don't know. Why is it an absolute directory path and not an dynamic directory path?

File: web-gui/docker-pyinstaller1/Dockerfile-py3-win32 (hunk 2 above)
[the remark this answers is not in the document]
Comment (vrlnx) [reply web-gui/docker-pyinstaller1/Dockerfile-py3-win32:1]: Look like the publisher did have haste typing this article

It is just `apt remove [package]` and `apt search [package]`
image

File: byob/crosscomp/build_environment.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (vrlnx) [reply byob/crosscomp/build_environment.sh:210]: Look like the publisher did have haste typing this article

It is just `apt remove [package]` and `apt search [package]`
image

[the remark this answers is not in the document]
Comment (vrlnx) [reply byob/crosscomp/build_environment.sh:210]: image

File: web-gui/docker-pyinstaller1/Dockerfile-py3-win32 (hunk 2 above)
[the remark this answers is not in the document]
Comment (malwaredllc) [reply web-gui/docker-pyinstaller1/Dockerfile-py3-win32:1]: I see, good catch!

File: byob/crosscomp/build_environment.sh (hunk 3 above)
[the remark this answers is not in the document]
Comment (malwaredllc) [reply byob/crosscomp/build_environment.sh:210]: Interesting, ok thanks for clarifying.

File: web-gui/startup.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (vrlnx) [reply web-gui/startup.sh:31]: Exclude this from the merge!

**Comment by malwaredllc** [comment]
Discussed with @vrlnx and I am going to decline this pull request and then he'll submit a new one shortly.

**Pull request closed without merging**